www/styles/globals.css

```diff
@@ -16,6 +16,12 @@ body {
   overflow: hidden;
 }
 
+h3 {
```

Comment:
Do we want the extra margin on all `h3`s.

Reply:
sorry that was a poor implementation of that style